### PR TITLE
Removed checks for BZs that have been closed for 2+ months

### PR DESCRIPTION
**Pull request by JacobCallahan**
These checks are consuming a significant amount of time, during test runs.
I also removed one test that is not valid.
See https://bugzilla.redhat.com/show_bug.cgi?id=1176600#c3

### CODE REVIEW

File: tests/foreman/cli/test_os.py [hunk 1]
```diff
@@ -15,12 +15,12 @@
 
 
 POSITIVE_CREATE_DATA = (
-    {'name': gen_string("latin1", 10)},
```
Comment (elyezer) [inline tests/foreman/cli/test_os.py]: Good catch

**Comment by elyezer** [comment]
ACK

File: tests/foreman/cli/test_os.py (hunk 1 above)
Comment (JacobCallahan) [inline tests/foreman/cli/test_os.py]: i am also going to go through the cli tests at some point and clean up a lot of the other mess.

**Comment by Ichimonji10** [comment]
ACK

File: tests/foreman/cli/test_os.py (hunk 1 above)
Comment (elyezer) [inline tests/foreman/cli/test_os.py]: Nice, looking forward it

**Pull request merged**